eq-author-api/src/validation/customKeywords/textLengthInRange.js

```diff
@@ -0,0 +1,35 @@
+module.exports = function(ajv) {
+  ajv.addKeyword("textLengthInRange", {
```

Comment:
I had a quick look but it might be a bit of a pain. The issue is Graphql stringifys JSON types so we can't use numeric validation on it and I couldn't find a non-messy place to do a type coercion.

Reply:
Might this help? https://ajv.js.org/coercion.html

Reply:
https://github.com/epoberezkin/ajv#coercing-data-types

Reply:
Yeah I just wasn't a fan of the validator modifying the underlying data but it's not too problematic.

Reply:
Yeah, I agree. We can just stick with this keyword then 👍